RELEASE-PROCEDURE.md

```diff
@@ -0,0 +1,472 @@
+
+   Clone the `dragonfly` repository.
+
+   ```bash
+   git clone git@github.com:dragonflyoss/dragonfly.git
+   cd dragonfly
+   ```
+
+   Create a new tag from the release branch.
+
+   ```bash
+   git checkout release-3.0
+   git tag v3.0.0
+   ```
+
+2. **Update the version in `version.go`**: Update the go version in `version.go` file.
+
+   ```go
+   hack/update-version-gorelease.sh
+   ```
+
+3. **Generate the ChangeLog**: Generate the changelog using the following command:
+
+   ```bash
+   make changelog
+   ```
+
+4. **Push the tag**: Push the tag to the remote repository using the following command:
+
+   ```bash
+   git push origin v3.0.0
+   ```
+
+5. **Wait for the CI/CD pipeline**: Wait for the CI/CD pipeline to complete. The pipeline will create the release artifacts, including binaries, Docker images, and other necessary files.
+
+Release pipeline will be triggered automatically when a new tag is pushed to the repository and release pipeline is configured in the `.github/workflows/release.yml` file.
+Release pipeline link is <https://github.com/dragonflyoss/dragonfly/actions/workflows/release.yml>, wait for the pipeline to complete and check the logs for any errors.
+If the pipeline finished, it will create a new release in the GitHub repository with the tag name and release notes.
+
+6. **Create a new release**: Create a new release in the GitHub repository following the steps below:
+
+Enter the [release page](https://github.com/dragonflyoss/dragonfly/releases) in the GitHub repository, and click the "edit" button to add the release notes
+and release the new version. Major release should not use the `git-chglog` to generate the release notes and need to be manually edited.
+
+7. **Archive the documentation of the old version**: Archive the documentation of the old version in the [d7y.io](https://github.com/dragonflyoss/d7y.io) repository.
+   The documentation should be archived in a separate directory with the version number.
+
+   Clone the `d7y.io` repository.
+
+   ```bash
+   git clone git@github.com:dragonflyoss/d7y.io.git
+   cd d7y.io
+   ```
+
+   Install npm dependencies.
+
+   ```bash
+   npm install
+   ```
+
+   Archive the documentation of the old version, parameter `v3.0.0` is the new release version.
+
+   ```bash
+   npm run version v3.0.0
+   ```
+
+   Push the changes to the `d7y.io` repository.
+
+   ```bash
+   git add .
+   git commit -m "chore: archive the documentation of v3.0.0"
+   git push origin main
+   ```
+
+   Merge the pull request to the `d7y.io` repository.
+
+8. **Update image version of Helm Chart**: Update the image version of [helm-charts](https://github.com/dragonflyoss/helm-charts) in the `charts` directory.
+   The image version should be updated to the new release version.
+
+   ```bash
+   git clone git@github.com:dragonflyoss/helm-charts.git
+   cd helm-charts
+   ```
+
+   Update the image version in the `charts/dragonfly/values.yaml` file.
```

Comment:
Here we need to clarify updates the manager and scheduler image tag, client and seedClient tag is not shipped with same version.